wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,68 @@
 # History of osu! 2010
+
+## January
+
+On the [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth, osu! had an arcade booth which featured a fully self-contained arcade machine running a touch-based version of osu! on it called osu!arcade. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
+
+![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")
+
+## February
+
+A new osu! update made small improvements in the overall user interface and added the ability to move all storyboard events by an arbitrary time value, allowing for more fine-tuned storyboard animations. In addition to this, usernames became clickable in chat logs.[^stable-b1460]
+
+## March
+
+Through February and March, [peppy](https://osu.ppy.sh/users/2) organised an art competition about drawing banners for in-game challenges. After a [community voting](https://osu.ppy.sh/community/forums/topics/27112?n=1) concluded, the osu! team picked the final selection. The winner's art being officially used as well as one year of [osu!supporter](/wiki/osu!supporter) or physical goods being awarded were in prospect for the competition's finalist.[^art-challenge]
+
+On March 8, an osu! update brought mainly performance and audio improvements to the game.[^stable-b1485]
+
+## April
+
+Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) was moved to the address `irc.ppy.sh`.[^irc]
+
+This month, peppy introduced a new star rating difficulty algorithm in an osu! update.[^stable-b1596]
+
+## August
+
+Since August 2, users became able to change their username once. To change their name, they had to have received an osu!supporter tag at least once beforehand, either through purchase or as a gift.[^name-change]
```

Comment:
advanced tenses are one of my weaknesses, but "users became able" sounds wrong to me really, so a tiny rewrite is in order
```suggestion
On August 2, all users had been granted a single username change. To change username, they had to had received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]
```

Reply:
"they had to had received" is not correct i'm pretty sure

```suggestion
On August 2, all users were granted a single username change. To change username, they had to have received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]
```

Reply:
thanks